histomicsui/rest/system.py

```diff
@@ -15,9 +15,9 @@
 #############################################################################
 
 import datetime
+import logging
```

Comment:
@zachmullen after looking at the differences between `girder-5` and `master`, it looks to me like we're no longer using `girder.logger` and instead importing `logging` and using `getLogger`. Is this correct?

Reply:
Yes, that's correct. I'd recommend reading through this comprehensive list of Girder 5's key changes: https://github.com/girder/girder/blob/v4-integration/docs/migration-guide.rst#3x--5x